lib/msprime.h

```diff
@@ -430,7 +432,8 @@ int msp_set_simulation_model_sweep_genic_selection(msp_t *self, double position,
 int msp_set_start_time(msp_t *self, double start_time);
 int msp_set_store_migrations(msp_t *self, bool store_migrations);
 int msp_set_store_full_arg(msp_t *self, bool store_full_arg);
-int msp_set_store_unary(msp_t *self, bool store_unary);
+int msp_set_additional_nodes(msp_t *self, int additional_nodes);
```

Comment:
Since the additional nodes argument is a flags value, we want it to be a fixed size and with an unsigned type (there can be confusion with the sign bit, otherwise). So, change the type here to ``uint32_t``.